### PR TITLE
Add sentry to neon broker

### PR DESCRIPTION
**Pull request by vadim2404**

### CODE REVIEW

File: charts/neon-proxy/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Kubernetes: `^1.18.x-x`
 | settings.authBackend | string | `"link"` | auth method used (console|link|postgres) |
 | settings.authEndpoint | string | `""` | auth endpoint, e.g. "http://console.neon/authenticate_proxy_request/" |
 | settings.domain | string | `""` | domain used in TLS cert for client postgres connections |
-| settings.sentryURL | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
+| settings.sentryUrl | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
```
Comment (lassizci) [inline charts/neon-proxy/README.md:63]: We now have a bit different conventions, as `neon-proxy` uses `sentryURL`, although probably not worth changing. And if changing, maybe using DSN would be the choice to consider to be consisten with the ENV variable and Sentry terminology. So just leaving the comment as a trail.

**Review by lassizci — APPROVED** [review]
It seems completing a bit incomplete feature rather than adding it from the scratch, so I would like to see that reflected in the PR descripiton (or title or commit message), but otherwise seems straight forward.

File: charts/neon-proxy/README.md (hunk 1 above)
Comment (vadim2404) [reply charts/neon-proxy/README.md:63]: I've made it similar: both containers accept `settings.sentryUrl` that will be transformed into `SENTRY_DSN` env variable. what do you propose to change in order to keep it clean?

Comment (lassizci) [reply charts/neon-proxy/README.md:63]: Ah, so I wasn't clear enough. The point of:
> So just leaving the comment as a trail.

Was actually not to propose a change, but leave a comment here for future code archeologists that might wonder :)


Indeed changing this to _DSN would require changing the existing values -files and I don't think we want that now.

**Pull request merged**